Core/CrossCuttingConcerns/Caching/ICacheService.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace WebNetSample.Core.CrossCuttingConcerns.Caching;
+
+public interface ICacheService
+{
+    void Add(string key, object value, int duration);
```

Comment:
Let's probably make it generic as well: `void Add<T>(string key, T value, int duration);`